File: SDKs/sportsdata.io/soccer-v3-scores/1.0/python/src/sdk/models/operations/upcomingschedulebyplayer.py
from dataclasses import dataclass, field
from typing import Any,Enum,List,Optional

class UpcomingScheduleByPlayerFormatEnum(str, Enum):
    XML = "xml"
    JSON = "json"


@dataclass
class UpcomingScheduleByPlayerPathParams:
    format: UpcomingScheduleByPlayerFormatEnum = field(default=None, metadata={'path_param': { 'field_name': 'format', 'style': 'simple', 'explode': False }})
    playerid: str = field(default=None, metadata={'path_param': { 'field_name': 'playerid', 'style': 'simple', 'explode': False }})
    

@dataclass
class UpcomingScheduleByPlayerRequest:
    path_params: UpcomingScheduleByPlayerPathParams = field(default=None)
    

@dataclass
class UpcomingScheduleByPlayerResponse:
    content_type: str = field(default=None)
    games: Optional[List[Any]] = field(default=None)
    status_code: int = field(default=None)
    

File: SDKs/sportsdata.io/soccer-v3-stats/1.0/go/pkg/models/operations/upcomingschedulebyplayer.go
package operations




type UpcomingScheduleByPlayerFormatEnum string

const (
    UpcomingScheduleByPlayerFormatEnumXML UpcomingScheduleByPlayerFormatEnum = "xml"
UpcomingScheduleByPlayerFormatEnumJSON UpcomingScheduleByPlayerFormatEnum = "json"
)


type UpcomingScheduleByPlayerPathParams struct {
    Format UpcomingScheduleByPlayerFormatEnum `pathParam:"style=simple,explode=false,name=format"`
    Playerid string `pathParam:"style=simple,explode=false,name=playerid"`
    
}

type UpcomingScheduleByPlayerRequest struct {
    PathParams UpcomingScheduleByPlayerPathParams 
    
}

type UpcomingScheduleByPlayerResponse struct {
    ContentType string 
    Games []interface{} 
    StatusCode int64 
    
}

